2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Configure stats pid to 4818
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/.config/wandb/settings
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/settings
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_init.py:_log_setup():492] Logging user logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_150342-bpgkcxf0/logs/debug.log
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_init.py:_log_setup():493] Logging internal logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_150342-bpgkcxf0/logs/debug-internal.log
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_init.py:init():532] calling init triggers
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-17 15:03:42,436 INFO    MainThread:4818 [wandb_init.py:init():554] re-initializing run, found existing run on stack: 075qhfma
2023-03-17 15:03:42,437 INFO    MainThread:4818 [wandb_run.py:_finish():1854] finishing run agnostic/test_run_pong/075qhfma
2023-03-17 15:03:42,438 ERROR   MainThread:4818 [jupyter.py:save_history():441] Run pip install nbformat to save notebook history
2023-03-17 15:03:42,438 INFO    MainThread:4818 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-03-17 15:03:42,438 INFO    MainThread:4818 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-03-17 15:03:42,438 INFO    MainThread:4818 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-17 15:03:42,438 INFO    MainThread:4818 [wandb_run.py:_restore():2066] restore
2023-03-17 15:03:42,438 INFO    MainThread:4818 [wandb_run.py:_restore():2072] restore done
2023-03-17 15:03:47,997 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-17 15:03:47,998 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-17 15:03:48,012 INFO    MainThread:4818 [wandb_run.py:_footer_sync_info():3398] logging synced files
2023-03-17 15:03:48,025 INFO    MainThread:4818 [wandb_init.py:init():588] starting backend
2023-03-17 15:03:48,025 INFO    MainThread:4818 [wandb_init.py:init():592] setting up manager
2023-03-17 15:03:48,034 INFO    MainThread:4818 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-17 15:03:48,038 INFO    MainThread:4818 [wandb_init.py:init():599] backend started and connected
2023-03-17 15:03:48,044 INFO    MainThread:4818 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-03-17 15:03:48,044 INFO    MainThread:4818 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-17 15:03:48,044 INFO    MainThread:4818 [wandb_init.py:init():687] updated telemetry
2023-03-17 15:03:48,098 INFO    MainThread:4818 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-03-17 15:03:49,400 INFO    MainThread:4818 [wandb_run.py:_on_init():2134] communicating current version
2023-03-17 15:03:49,482 INFO    MainThread:4818 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-17 15:03:49,482 INFO    MainThread:4818 [wandb_init.py:init():775] starting run threads in backend
2023-03-17 15:03:49,576 INFO    MainThread:4818 [wandb_run.py:_console_start():2114] atexit reg
2023-03-17 15:03:49,577 INFO    MainThread:4818 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-03-17 15:03:49,577 INFO    MainThread:4818 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-03-17 15:03:49,578 INFO    MainThread:4818 [wandb_run.py:_redirect():2059] Redirects installed.
2023-03-17 15:03:49,579 INFO    MainThread:4818 [wandb_init.py:init():817] run started, returning control to user process
2023-03-17 15:04:22,104 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Configure stats pid to 4818
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/.config/wandb/settings
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/settings
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_init.py:_log_setup():492] Logging user logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_150422-8ejhexe9/logs/debug.log
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_init.py:_log_setup():493] Logging internal logs to /home/kidimerek/Documents/Studium/Thesis/agnostic_rl-main/agent/wandb/run-20230317_150422-8ejhexe9/logs/debug-internal.log
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f167cb61880>
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_init.py:init():532] calling init triggers
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2023-03-17 15:04:22,105 INFO    MainThread:4818 [wandb_init.py:init():554] re-initializing run, found existing run on stack: bpgkcxf0
2023-03-17 15:04:22,107 INFO    MainThread:4818 [wandb_run.py:_finish():1854] finishing run agnostic/test_run_pong/bpgkcxf0
2023-03-17 15:04:22,107 INFO    MainThread:4818 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-03-17 15:04:22,107 INFO    MainThread:4818 [wandb_run.py:_restore():2066] restore
2023-03-17 15:04:22,108 INFO    MainThread:4818 [wandb_run.py:_restore():2072] restore done
2023-03-17 15:04:29,106 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-03-17 15:04:29,106 INFO    MainThread:4818 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-03-17 15:04:29,118 INFO    MainThread:4818 [wandb_run.py:_footer_sync_info():3398] logging synced files
